src/projectGeneration/templates.test.ts

```diff
@@ -103,5 +130,30 @@ describe("templates.ts", () => {
       const result = await pickTemplate([template]);
       assert.strictEqual(result, undefined);
     });
+
+    it("should return templates sorted alphabetically by name", () => {
```

Comment:
This test has .... issues:
  1. `unsortedTemplates` is ... already sorted.
  2. tests .... a function defined right inlined in the test.